README.md

```diff
@@ -220,17 +189,22 @@ resource information in `resource.json` will be injected to the template using
   }
 }
 ```
-_Sample `marathon.json.mustache`._
 
 See the
 [Marathon API Documentation](https://mesosphere.github.io/marathon/docs/rest-api.html)
 for more detailed instruction on app definitions.
 
 #### `command.json`
+|Packaging Version|   |
+|-----------------|---|
+|2.0|optional|
+|3.0|optional **[Deprecated]**|
 
-This file is **optional**. Describes how to install the package's CLI.
-Currently the only supported format is a Pip requirements file where each
-element in the array is a line in the requirements file.
+As of `packagingVersion` `3.0`, `command.json` is deprecated in favor of the `.cli` property of `resources.json`.
+See [CLI Resources](#cli-resources) for details.
+
+Describes how to install the package's pip CLI. This document represents the format of a Pip requirements file where
```

Comment:
"package's pip CLI." -> "package's CLI via pip, the Python package manager."

Or perhaps just "package's CLI via [pip](https://en.wikipedia.org/wiki/Pip_(package_manager))."?